gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds
+from ...data import DataStore, ObservationList
+
+
+@requires_data('gammapy-extra')
+def test_make_psftable():
```

Comment:
@cdeil 
ok but I mean shouldn't we try to undersand why the gaussian psf are so different for these two runs?